README.md

```diff
@@ -224,6 +224,10 @@ config.json:
 
 Note, that you must have environment variables `AZURE_STORAGE_ACCOUNT` and `AZURE_STORAGE_ACCESS_KEY`, or `AZURE_STORAGE_CONNECTION_STRING` defined for authentication.
 
+If you wish to provide custom authentication keys for specific environments you can provide it also via the keys `azureConnectionString`, `azureAccount` or `azureAccessKey`.
+
+**Its not recommended to put authentication keys in code. Always provide them via environment variables.**
+
 config.json:
 
 ```json
```

Comment:
The formatting of this looks bad now:

<img width="801" alt="image" src="https://user-images.githubusercontent.com/5524384/84052190-76547f80-a96d-11ea-906f-de268396f625.png">


To fix, change it to be javascript:

```suggestion
```js
```

Also, for js config files, we should add `module.exports = ` before the object.